python-api/app/main_api.py

```diff
@@ -0,0 +1,80 @@
+from fastapi import FastAPI, HTTPException
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.responses import FileResponse
+import duckdb
+import os
+import json
+from shapely.geometry import shape
+import geopandas as gpd
+import pandas as pd
+import pygadm as pg
+
+app = FastAPI()
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+ddb = duckdb.connect()
+ddb.execute("SET home_directory='/app'") 
+ddb.install_extension("spatial")
+ddb.load_extension("spatial")
+ddb.install_extension("httpfs")
+ddb.load_extension("httpfs")
+
+
+countries_parquet = "https://data.fieldmaps.io/adm0/osm/intl/adm0_polygons.parquet"
+regions_parquet = "https://data.fieldmaps.io/edge-matched/humanitarian/intl/adm1_polygons.parquet"
+
+if os.path.exists("/app/countries.json") == False:
+    countries=ddb.sql("SELECT adm0_src, adm0_name, geometry_bbox FROM read_parquet('%s')" % countries_parquet).df()
+    with open('/app/countries.json', 'w') as f:
```

Comment:
fetched here... (see previous comment)

Reply:
https://github.com/GEO-BON/bon-in-a-box-pipeline-engine/pull/313#issuecomment-3618715714